src/stan/math/prim/mat/prob/multi_normal_rng.hpp

```diff
@@ -20,10 +20,10 @@ namespace stan {
   namespace prob {
     using Eigen::Dynamic;
 
-    template <class RNG>
+    template <class RNG, typename T1, typename T2>
```

Comment:
Yes.

On Wed, Mar 11, 2015 at 8:18 PM, Bob Carpenter <notifications@github.com>
wrote:

> In src/stan/math/prim/mat/prob/multi_normal_rng.hpp
> <https://github.com/stan-dev/stan/pull/1210#discussion_r26269055>:
>
> > @@ -20,10 +20,10 @@ namespace stan {
> >    namespace prob {
> >      using Eigen::Dynamic;
> >
> > -    template <class RNG>
> > +    template <class RNG, typename T1, typename T2>
>
> We only allow RNGs to apply to double values because we can't autodiff
> through them, so I'm guessing Daniel is suggesting to just hard-code double
> as the scalar type. - Bob
> … <#14c0b597959d8fbc_>
>  On Mar 12, 2015, at 8:44 AM, Joshua Pritikin <notifications@github.com>
> wrote: In src/stan/math/prim/mat/prob/multi_normal_rng.hpp: > @@ -20,10
> +20,10 @@ namespace stan { > namespace prob { > using Eigen::Dynamic; > > -
> template <class RNG> > + template <class RNG, typename T1, typename T2> How
> so? How do you make a function accept MatrixBase arguments without using
> template parameters? I'm baffled. — Reply to this email directly or view it
> on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1210/files#r26269055>.
>